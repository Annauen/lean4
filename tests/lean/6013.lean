def test (a : String) : String :=
  let x : String â†’ String := _
  have : x = 10 := by rfl
