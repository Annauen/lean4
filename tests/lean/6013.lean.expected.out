6013.lean:4:0: error: unexpected end of input
6013.lean:2:29-2:30: error: don't know how to synthesize placeholder
context:
a : String
⊢ String → String
